### ISSUE TITLE
HR-employee gets notification after DI-creation

### ISSUE THREAD

**Issue by veitmo** [amosproj/amos2022ss04-digital-identity#17]
## User story
1. As a {HR employee}
2. I want / need {to get feedback on DIdentity frontend as soon as the digital identity has been created and is usable}
3. So that {I have an overview over success and that I can detect failures easily}

## Acceptance criteria
* after clicking on the "create" button on the DI frontend, a success/error message is displayed (e.g. at the top of the page)
* success message with background color green
* error message with background color red
* success means that the DI has been successfully created on Lissi-API and email with QR-code is sent

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
